source/backend/api/Models/Concepts/FormDocument/FormDocumentTypeModel.cs

```diff
@@ -0,0 +1,28 @@
+namespace Pims.Api.Models.Concepts
+{
+    /// <summary>
+    /// FormDocumentTypeModel class, provides a model to represent form document types.
+    /// </summary>
+    public class FormDocumentTypeModel : BaseAppModel
+    {
+        /// <summary>
+        /// get/set - Form type code.
+        /// </summary>
+        public string FormTypeCode { get; set; }
+
+        /// <summary>
+        /// get/set - Form type document id.
```

Comment:
These comments are strictly optional in my mind, in case you want to omit them in the future. I think we really only need comments when a field is doing something unusual.